antora-ui-camel/src/css/vars.css

```diff
@@ -130,10 +130,10 @@
   --nav-panel-height: calc(var(--nav-height) - var(--drawer-height));
   --nav-panel-height--desktop: calc(var(--nav-height--desktop) - var(--drawer-height));
   --nav-width: calc(270 / var(--rem-base) * 1rem);
-  --doc-max-width: calc(720 / var(--rem-base) * 1rem);
-  --doc-max-width--desktop: calc(828 / var(--rem-base) * 1rem);
-  --static-max-width: calc(720 / var(--rem-base) * 1rem);
-  --static-max-width--desktop: calc(1140 / var(--rem-base) * 1rem);
+  --doc-max-width: 81vw;
+  --doc-max-width--desktop: 64vw;
+  --static-max-width: 81vw;
+  --static-max-width--desktop: 88vw;
```

Comment:
Previously we had this based on screen resolutions and the rem unit, not sure where these numbers come from, can you explain them?